orc8r/cloud/go/services/configurator/mconfig/marshal.go

```diff
@@ -27,7 +28,7 @@ func MarshalConfigs(configs map[string]proto.Message) (ConfigsByKey, error) {
 	for k, v := range configs {
 		anyVal, err := ptypes.MarshalAny(v)
 		if err != nil {
-			return nil, errors.WithStack(err)
```

Comment:
We have no replacement for `errors.WithStack`?

Reply:
No, you still get the file name and line but no call stack.